APMrover2/system.cpp

```diff
@@ -332,6 +332,9 @@ bool Rover::arm_motors(AP_Arming::ArmingMethod method)
     // Set the SmartRTL home location. If activated, SmartRTL will ultimately try to land at this point
     g2.smart_rtl.set_home(true);
 
+    // initialize simple mode heading
```

Comment:
We should start to think about adding `#ifdef`s around new things like this, so we don't have the small upheaval we had in Copter to make large slabs optional.